apps/root/src/pages/strategy-guardian-detail/strategy-management/withdraw/tx-manager/index.tsx

```diff
@@ -37,8 +42,41 @@ const EarnWithdrawTransactionManager = ({ strategy, setHeight }: EarnWithdrawTra
 
   return (
     <>
+      <Modal
```

Comment:
This should be on a different component

Reply:
Also isnt this modal missing the expected changes?

Reply:
Yes, the modal should do an sdk call to calculate the market swap. For this task i did a quick modal to make the flow work, and I have a task specifically for tackle this. Sorry didn't mentioned it!